addon/bootstrap.js

```diff
@@ -77,7 +93,21 @@ function shouldDisable() {
   return getBoolPref(USER_DISABLE_PREF) || getBoolPref(SYSTEM_DISABLE_PREF);
 }
 
+// the postmessage channel opened once the webextension is started
+let webExtensionPort;
+function onConnect(port) {
+  webExtensionPort = port;
+}
+
+function onClick() {
+  if (!webExtensionPort) { return; } // just do nothing till the port is ready
+  // TODO: figure out how to send the tab ID over
+  webExtensionPort.postMessage({ content: "click" });
+}
+
+
 function handleStartup() {
+  // TODO: check the pref _before_ calling into ExtensionsUtils.
```

Comment:
this is an older todo, so it might make sense to fix directly in master. filed #2879